src/components/ConfirmationModal/_confirmation-modal.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../scss/functions/z-index";
+
+#confirmation-modal-container {
+  .p-modal {
+    align-items: flex-end;
+    justify-content: right;
+    z-index: z("zelda");
+
+    h5 {
+      margin-bottom: 0;
+      padding-top: 0 !important;
+    }
+
+    pre {
+      // reset the vanilla pre styling
+      background-color: inherit;
+      border: none;
+      padding: 0;
+      // extra styles
+      white-space: pre-wrap;
+      word-break: keep-all;
+    }
+
+    &__dialog {
+      max-width: 522px;
```

Comment:
What is this magic number?

Reply:
Heh yes pretty magical, it was the max-width set in the design :)